Makefile

```diff
@@ -31,22 +27,23 @@ OPENSTACK_CR         ?= ${OPERATOR_BASE_DIR}/openstack-operator/${OPENSTACK_CTLP
 OPENSTACK_BUNDLE_IMG ?= quay.io/openstack-k8s-operators/openstack-operator-bundle:latest
 
 # Infra Operator
-INFRA_IMG        ?= quay.io/openstack-k8s-operators/infra-operator-index:latest
-INFRA_REPO       ?= https://github.com/openstack-k8s-operators/infra-operator.git
-INFRA_BRANCH     ?= master
+INFRA_IMG           ?= quay.io/openstack-k8s-operators/infra-operator-index:latest
+INFRA_REPO          ?= https://github.com/openstack-k8s-operators/infra-operator.git
+INFRA_BRANCH        ?= master
 
 # Memcached
+# MEMCACHED_IMG     ?= (tis is unused because this is part of infra operator)
 MEMCACHED           ?= config/samples/memcached_v1beta1_memcached.yaml
 MEMCACHED_CR        ?= ${OPERATOR_BASE_DIR}/infra-operator/${MEMCACHED}
-MEMCACHED_IMG       ?= ${SERVICE_REGISTRY}/${SERVICE_ORG}/openstack-memcached:current-tripleo
+MEMCACHED_DEPL_IMG  ?= unused
```

Comment:
wondering if it is confusing if we name the service image in some services `<>_DEPL_IMG` and in others `_IMG`, e.g. `KEYSTONEAPI_IMG` even we have there the `API` part. maybe we should have them all the same extension, either `_DEPL_IMG` or `_SVC_IMG`

Reply:
Yeah I agree with it. Initially I was thinking of renaming parameters for operator images but that might cause more impact now as these are intensively used by CI. So I've updated the ones used for services.

Note that I selected _DEPL_IMG because that is already used and also the "service" term can be confusing because of the same term in k8s.